Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ServerGameNetPortal.cs

```diff
@@ -201,53 +204,42 @@ void ApprovalCheck(byte[] connectionData, ulong clientId, NetworkManager.Connect
             if (m_Portal.NetManager.ConnectedClientsIds.Count >= CharSelectData.k_MaxLobbyPlayers)
             {
                 gameReturnStatus = ConnectStatus.ServerFull;
-                //TODO-FIXME:Netcode Issue #796. We should be able to send a reason and disconnect without a coroutine delay.
-                //TODO:Netcode: In the future we expect Netcode to allow us to return more information as part of
-                //the approval callback, so that we can provide more context on a reject. In the meantime we must provide the extra information ourselves,
-                //and then manually close down the connection.
-                SendServerToClientConnectResult(clientId, gameReturnStatus);
-                SendServerToClientSetDisconnectReason(clientId, gameReturnStatus);
-                StartCoroutine(WaitToDisconnect(clientId));
-                return;
             }
-
-            string payload = System.Text.Encoding.UTF8.GetString(connectionData);
-            var connectionPayload = JsonUtility.FromJson<ConnectionPayload>(payload); // https://docs.unity3d.com/2020.2/Documentation/Manual/JSONSerialization.html
-
-            int clientScene = connectionPayload.clientScene;
-
-            Debug.Log("Host ApprovalCheck: connecting client with player ID: " + connectionPayload.playerId);
-
-            gameReturnStatus = SessionManager<SessionPlayerData>.Instance.SetupConnectingPlayerSessionData(clientId, connectionPayload.playerId,
-                new SessionPlayerData(clientId, connectionPayload.playerName, m_Portal.AvatarRegistry.GetRandomAvatar().Guid.ToNetworkGuid(), 0, true))
-                ? ConnectStatus.Success
-                : ConnectStatus.LoggedInAgain;
-
-            //Test for Duplicate Login.
-            if (gameReturnStatus == ConnectStatus.LoggedInAgain)
+            else
             {
-                SessionPlayerData? sessionPlayerData =
-                    SessionManager<SessionPlayerData>.Instance.GetPlayerData(connectionPayload.playerId);
-
-                ulong oldClientId = sessionPlayerData?.ClientID ?? 0;

```

Comment:
What happened to this logic?

Reply:
We discussed it a while ago while I was working on #482 . We want to deny the new connection, not kick the first one. Anyways, this code did not work, it sent a disconnect reason to the old client, but disconnected the new one, so the old client didn't do anything with the disconnect reason message and the new denied one did not know why they were disconnected

Reply:
oh ok then, I thought this was addressed already. ok!

Reply:
It is, but 482 is still in review since it won't be needed for GDC